pages/guides/use.md

```diff
@@ -27,7 +27,7 @@ Use the following tags to structure your Living Style Guide:
       <td>Declare a stylesheet file as page in your site.</td>
   </tr>
   <tr>
-      <td>@@style</td>
+      <td>@@styles</td>
```

Comment:
Looks like that must be necessary in the `.md` and it gets collapsed to a single `@` in the [generated page](http://documentcss.com/docs/use.html)?